packages/Heading/Heading.modules.scss

```diff
@@ -1,6 +1,7 @@
 @import '~@tds/core-colours/colours';
 @import '../../shared/scss/typography';
 @import '../../shared/scss/responsive';
+@import '../../packages/Responsive/scss/responsive.scss';
```

Comment:
This should import responsive.scss from the core-responsive package, so that core-responsive is declared as a dependency. Notice line 1.

It should be something like `@import '~@tds/core-responsive/responsive';`